deploy/template.yaml

```diff
@@ -143,7 +143,7 @@ Mappings:
       LOGSLEVEL: "debug"
       SUPPORTTRIAGEPAGE: "1"
       WEBCHATSOURCEURL: "https://uat-chat-loader.smartagent.app/loader/main.js"
-      SUPPORTWEBCHATCONTACT: "0"
+      SUPPORTWEBCHATCONTACT: "1"
```

Comment:
Are we leaving webchat on in DEV going forward?  It won't be prod like if we do.

Reply:
Hi @alex9smith , as per the questions above from @andrew-moores , would like your input here pls?

Reply:
I think we should switch this back off in dev - we know we're not going to be working on the webchat for at least this sprint. Dev isn't going to be identical to production all the time but we should try and keep it as close as possible for longer-term things.